project-base/storefront/components/Pages/Search/SearchProducts.tsx

```diff
@@ -0,0 +1,95 @@
+import { SearchProductsContent } from './SearchProductsContent';
+import { useSearchProductsData } from './helpers';
+import { FilterIcon } from 'components/Basic/Icon/IconsSvg';
+import { FilterPanel } from 'components/Blocks/Product/Filter/FilterPanel';
+import { SortingBar } from 'components/Blocks/SortingBar/SortingBar';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { ProductOrderingModeEnumApi } from 'graphql/generated';
+import { getInternationalizedStaticUrls } from 'helpers/getInternationalizedStaticUrls';
+import useTranslation from 'next-translate/useTranslation';
+import dynamic from 'next/dynamic';
+import { useRef, useState } from 'react';
+import { twJoin } from 'tailwind-merge';
+
+const Overlay = dynamic(() => import('components/Basic/Overlay/Overlay').then((component) => component.Overlay));
+
+export const SearchProducts: FC = () => {
+    const { t } = useTranslation();
+    const paginationScrollTargetRef = useRef<HTMLDivElement>(null);
+    const { url } = useDomainConfig();
+    const [searchUrl] = getInternationalizedStaticUrls(['/search'], url);
+    const [isPanelOpen, setIsPanelOpen] = useState(false);
+
+    const { searchProductsData, fetching, loadMoreFetching } = useSearchProductsData();
+
+    const handlePanelOpenerClick = () => {
+        const body = document.getElementsByTagName('body')[0];
+
+        setIsPanelOpen((prev) => {
+            const newValue = !prev;
+            body.style.overflow = newValue ? 'hidden' : 'visible';
+
+            return newValue;
+        });
+    };
+
+    if (!searchProductsData) {
+        return null;
+    }
+
+    return (
+        <>
+            <div className="mt-6">
+                <div className="h3 mb-3">{t('Found products')}</div>
```

Comment:
You have simplified this title quite heavily. We lost the info about pages and I am not sure if that is desired. Can you confirm that it is OK and based on what?